src/QuoteBundle/Tests/Listener/Mailer/QuotePdfListenerTest.php

```
@@ -28,28 +27,28 @@
 class QuotePdfListenerTest extends TestCase
 {
-    use MockeryPHPUnitIntegration;
-
     public function testListener(): void
     {
         $quote = new Quote();
 
-        $mailer = M::mock(MailerInterface::class);
-        $mailer->shouldReceive('send');
+        /** @var MailerInterface&MockObject $mailer */
+        $mailer = $this->createMock(MailerInterface::class);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$mailer'. (undefined)

(UnusedLocalVariable)